protobuf/scalapb/scalapb.proto

```diff
@@ -291,6 +291,18 @@ message EnumOptions {
   // must be present.
   optional string type = 3;
 
+  // Custom annotations to add to the generated case object.
+  repeated string annotations_for_trait = 4;
```

Comment:
For consistency with the existing `companion_annotations` name, please rename to`trait_annotations` - same comment apply to the new fields below.